docs/source/guide/zne-3-options.md

```diff
@@ -216,6 +216,66 @@ folding method. Fidelity values must be between zero and one.
 
 +++
 
+#### Folding gates by layer
+
+Circuit debugging techniques (as proposed in
+[arXiv:2204.06056](https://arxiv.org/abs/2204.06056)) utilize the ability to
+apply layerwise folding of a circuit.
+
+A quantum circuit $C$ may be represented as a series of $d$ layers. Each layer
+$L$ contains one or more quantum gates acting concurrently on an $n$-qubit
+system:
+
+$$C = L_d L_{d-1} \cdots L_2 L_1$$.
+
+The $i$-inverse operation of a circuit $C$ is defined as
```

Comment:
Something is off here:
<img width="516" alt="latex-bug-mitiq" src="https://user-images.githubusercontent.com/14573436/230585635-c9656705-328f-4a35-ba8e-76c493ed3d7a.png">
Note you can preview the docs relative to this PR by clicking on the CI check for readthedocs. Direct link [here](https://mitiq--1767.org.readthedocs.build/en/1767/).

Reply:
Hmm. Oddly enough, this is rendering fine for me locally and I'm not seeing any obvious LaTeX issues with the source. Maybe on the next build it will fix itself. Will see.

Reply:
Resolved. Thanks again @natestemen